Upgrade-Guide/op_version.md

```diff
@@ -0,0 +1,29 @@
+
+###op-version
+op-version is the operating version of the Gluster which is running.
+
+After Gluster upgrade, it is important that op-version is updated in all peers to avoid any 
+compatibility issues.
+
+###Updating op-version
+
+Once Gluster is upgraded, ensure that the op-version is also updated.
+
+    [root@~]#cat /var/lib/glusterd/vols/tv1/info  | grep ^op-version
+
+
+If this is not done automatically set op-version as below.
+For example, after upgrading to glusterfs-3.7.1, set op-version as:
+
+    [root@~]#gluster volume set all cluster.op-version 30701
+
+### Understanding op-version
+
+The op-version for a given X.Y.Z release will be an integer XYZ, with
```

Comment:
@SaravanaStorageNetwork That's not true as of now. If any new release has introduced a feature/volume tunables which demand a new op-version, then only its added. I believe your question was can we increment this op-version with every releases to keep the equation simple. @kshlm what's your view point here?